apps/bond/components/BondingProducts/Bonding/BondingList/BondingList.jsx

```diff
@@ -223,8 +233,16 @@ const getColumns = (onClick, isActive, acc, depositoryAddress, hideEmptyProducts
   return columns;
 };
 
-const sortList = (list) =>
+const sortProducts = (list) =>
   list.sort((a, b) => {
+    // if the current price of the LP token is zero, then move it to the end of the list
+    // NOTE: It can be zero because
+    // - the API returns zero (shouldn't happen) OR
+    // - has error OR
+    // - not fetched yet
```

Comment:
is this our own API?
if so it should return some nullish reponse rather than 0, as this is an incorrect value..
no worries if not